python/pysmurf/client/util/smurf_util.py

```diff
@@ -2270,8 +2270,8 @@ def channel_to_freq(self, band, channel=None, yml=None):
         ----
         band : int
             The band the channel is in.
-        channel : int or None, optional, default none
-            The channel number.
+        channel : int, None, or array
```

Comment:
Sorry, just to hold to docstring convention, could you just change to `channel : int, None, or array, optional, default None`?